.github/PULL_REQUEST_TEMPLATE/release_pr_template.md

```diff
@@ -0,0 +1,6 @@
+
+
+# Review Checklist
+
+- [ ] Writes a summary in `whatsnew/whatsnew-pt-PT` of the new changes
+- [ ] Terms and conditions reflect the changes
```

Comment:
Terms and conditions should be updated on every PR, if necessary. The reason is the app will be used by beta testers, which are also subject to the terms and conditions of uni.

Reply:
I think it can be a little annoying and pushing the responsibility to the person who made the release and its reviews, will allow taking a preoccupation from each PR

Reply:
I agree that it is a bit annoying but, with each PR, we are essentially creating a release (for our beta testers). As such, the terms and conditions need to be kept up-to-date for them. If we remove public access from the beta track, we can push this to the release checklist.